auth_signup_partner_company/tests/test_auth_signup_partner_company.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestAuthSignupPartnerCompany(TransactionCase):
+    def setUp(self):
+        super(TestAuthSignupPartnerCompany, self).setUp()
+        self.env["ir.config_parameter"].set_param("auth_signup.invitation_scope", "b2c")
+        self.company = self.env.company
+        self.company_jpy = self.env["res.company"].create(
+            {
+                "name": "Japan company",
+                "currency_id": self.env.ref("base.JPY").id,
+                "country_id": self.env.ref("base.jp").id,
+            }
+        )
```

Comment:
```suggestion
        self.company_1 = self.env.company
        self.company_2 = self.env["res.company"].create({"name": "company 2"})
```

Let's remove the attributes not used in the test.